.cspell.json

```diff
@@ -136,6 +136,7 @@
     {
       "filename": "**/{*.cpp,*.hpp,*.py,*.sh,*.xml,*.yaml,CMakeLists.txt}",
       "ignoreRegExpList": [
+        "(max|min)(xyz)",
```

Comment:
It's a good solution, but I think we have to encourage people to replace `minx` with `min_x`.
So I'm sorry, but could you close this PR and leave the min/max typos as they are? :pray:

Reply:
Ok!